web/api/v1/api.go

```diff
@@ -1045,7 +1045,16 @@ type Alert struct {
 }
 
 func (api *API) alerts(r *http.Request) apiFuncResult {
-	alertingRules := api.rulesRetriever(r.Context()).AlertingRules()
+	if err := r.ParseForm(); err != nil {
```

Comment:
Let's kill it for now, to keep discussion simpler I guess

Reply:
Yes, have removed this!